bin/http-server

```diff
@@ -80,6 +82,54 @@ else if (colors) {
     request: function () {}
   };
 }
+if (superverbose) {
```

Comment:
Since this is an expanded version of the already-existing `logger.request`, could we move this inside the `if` block on line 57, rather than define a function and then immediately override it?